app/assets/stylesheets/application.css.scss

```diff
@@ -49,6 +49,9 @@ body {
   text-align: center;
 }
 
+.link-to-text-color {
+  color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.